generator/vogtle_data_generator.py

```diff
@@ -58,49 +58,120 @@ def __init__(self, directory=None):
         if directory:
             self.directory = directory
 
-    def generate_inspections(self, rows):
+    @staticmethod
+    def generate_itaac_effort_str(itaac_id, effort_id,
+                                  effort_type, effort_range):
+        """Generate string, bar seperated value of itaac_effort entry
```

Comment:
- [ ] `seperated` should be `separated`

```suggestion
        """Generate string, bar separated value of itaac_effort entry
```